src/validation/admin/team-category-schema/team-category-schema.ts

```diff
@@ -0,0 +1,36 @@
+import { TEAM_CATEGORY_VALIDATION } from '../../../const/admin/team';
+import * as Yup from 'yup';
+
+export const TeamCategoryValidationSchema = Yup.object({
+    name: Yup.string()
+        .trim()
+        .required(TEAM_CATEGORY_VALIDATION.name.getRequiredError)
+        .min(TEAM_CATEGORY_VALIDATION.name.min, TEAM_CATEGORY_VALIDATION.name.getMinError())
+        .max(TEAM_CATEGORY_VALIDATION.name.max, TEAM_CATEGORY_VALIDATION.name.getMaxError()),
+
+    description: Yup.string()
+        .trim()
+        .required(TEAM_CATEGORY_VALIDATION.description.getRequiredError)
+        .min(TEAM_CATEGORY_VALIDATION.description.min, TEAM_CATEGORY_VALIDATION.description.getMinError())
+        .max(TEAM_CATEGORY_VALIDATION.description.max, TEAM_CATEGORY_VALIDATION.description.getMaxError()),
+});
+
+export const TEAM_CATEGORY_VALIDATION_FUNCTIONS = {
+    validateName: (value: string): string | undefined => {
+        try {
+            TeamCategoryValidationSchema.validateSyncAt('name', { name: value });
+            return undefined;
+        } catch (error: any) {
+            return error.message;
+        }
+    },
+
+    validateDescription: (value: string): string | undefined => {
+        try {
+            TeamCategoryValidationSchema.validateSyncAt('description', { description: value });
+            return undefined;
+        } catch (error: any) {
+            return error.message;
+        }
+    },
+};
```

Comment:
Check my comment with Factory for this https://github.com/ita-social-projects/VictoryCenter-Client/pull/69/files#r2470453427
```suggestion
export const TEAM_CATEGORY_VALIDATION_FUNCTIONS = {
    validateName: createValidator(TeamCategoryValidationSchema, 'name'),
    validateDescription: createValidator(TeamCategoryValidationSchema, 'description'),
};
```

Reply:
I think it would be a better decision to make a separate ticket with migration for such approach everywhere